tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -283,7 +283,24 @@ def run(self):
                     _cluster = _cluster.load()
                     _cluster.status = ""
                     _cluster.last_sync = str(tendrl_now())
-                    _cluster.is_managed = "yes"
+                    # Mark the first sync done flag
+                    try:
+                        NS._int.client.read(
+                            "/clusters/%s/nodes/%s/first_sync_done" %
+                            (
```

Comment:
Please add this as attribute to object  https://github.com/Tendrl/commons/blob/master/tendrl/commons/objects/cluster_node_context/__init__.py